src/main/java/mate/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,246 @@
+package mate.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import mate.lib.Dao;
+import mate.lib.exception.DataProcessingException;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.model.Manufacturer;
+import mate.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    private static final int ZERO_PLACEHOLDER = 0;
+    private static final int SHIFT = 2;
```

Comment:
What exactly shift? Let's added one more word to this constant name that will describe what this shift does